cryptol-ast

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+CRYPTOL=/Users/emullen/cryptol/dist/build/Cryptol/cryptol
```

Comment:
I am not sure that we need this file, but if we have it should not contain hard-coded local paths.